cmd/notation/manifest.go

```diff
@@ -43,27 +28,47 @@ func getManifestDescriptorFromReference(ctx context.Context, opts *SecureFlagOpt
 	return repo.Resolve(ctx, ref.ReferenceOrDefault())
 }
 
-func getManifestDescriptorFromFile(path, mediaType string) (notation.Descriptor, error) {
-	file, err := os.Open(path)
-	if err != nil {
-		return notation.Descriptor{}, err
-	}
-	defer file.Close()
-	return getManifestDescriptorFromReader(file, mediaType)
-}
+// func getManifestDescriptorFromContext(ctx context.Context, opts *RemoteFlagOpts, ref string) (notation.Descriptor, error) {
```

Comment:
Do you want to clean up this code instead of commenting out? BTW, do you want to clean flags like `--local` in `common.go`?